telematic_system/telematic_apps/web_app/server/server.env

```diff
@@ -13,4 +13,25 @@ DB_PORT=3307
 # Security
 GRAFANA_HTPASSWORD=/opt/apache2/grafana_htpasswd
 SESSION_TIMEOUT=3600
-SECRET=my_secret_telematic_ui
\ No newline at end of file
+SECRET=my_secret_telematic_ui
+
+# File upload settings
+### Destination: HOST or S3
+### If S3, files are uploaded to the s3 bucket. Otherwise, files are uploaded to HOST machine with destination path
+UPLOAD_DESTINATION=s3
+UPLOAD_DESTINATION_PATH=/opt/telematics/upload
+
+# S3 bucket credentials
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_KEY=
+S3_BUCKET=
+S3_REGION=us-east-2
+
+# File upload service
+UPLOAD_HTTP_PORT = 9011
+UPLOAD_TIME_OUT=3600000 # Milliseconds
+UPLOAD_MAX_FILE_SIZE=21474836480 #20 GB
+
+# NATS config
+NATS_SERVERS = localhost:4222
+FILE_PROCESSING_SUBJECT=ui.file.procressing
```

Comment:
Can we add a comment here that changing this would require an update on the processing service side where we are listening on this subject

Reply:
Addressed https://github.com/usdot-fhwa-stol/cda-telematics/pull/174/commits/116d11d37074db7071f7d070a1f25a6865de71b6